packages/remix/src/index-platform.ts

```diff
@@ -0,0 +1,28 @@
+// This file should only export "server" exports that are implemented by your
+// platform and thus require built-in implementations such as:
+//  - `node:async_hooks`
+//  - `node:fs`
+//  - `node:zlib`
+//
+// All other APIs based on core Javascript/Web Fetch API should be exported
+// from the root remix package in `index.ts`
+
+export { asyncContext, getContext } from './lib/async-context-middleware'
```

Comment:
Just a note on import/export paths from files (not packages): we always use the file extension to avoid ambiguity. So this should be:


```suggestion
export { asyncContext, getContext } from './lib/async-context-middleware.ts'
```

Maybe we want to add an eslint rule that catches this?

Reply:
oh interesting - I thought that would already fail at build time and didn't even notice the LLM left them out.  I'll look into a way to catch this automatically 👍